src/main/scala/no/ndla/conceptapi/controller/DraftConceptController.scala

```diff
@@ -10,59 +10,40 @@ package no.ndla.conceptapi.controller
 import com.typesafe.scalalogging.LazyLogging
 import no.ndla.conceptapi.ConceptApiProperties
 import no.ndla.conceptapi.auth.User
-import no.ndla.conceptapi.model.api.{
-  Concept,
-  ConceptSearchParams,
-  ConceptSearchResult,
-  Error,
-  NewConcept,
-  NotFoundException,
-  SubjectTags,
-  TagsSearchResult,
-  UpdatedConcept,
-  ValidationError
-}
-import no.ndla.conceptapi.model.domain.{Language, SearchResult, Sort}
+import no.ndla.conceptapi.model.api.{Concept, ConceptSearchParams, ConceptSearchResult, NotFoundException}
+import no.ndla.conceptapi.model.domain.{ConceptStatus, Language, SearchResult, Sort}
 import no.ndla.conceptapi.model.search.SearchSettings
-import no.ndla.conceptapi.service.search.{ConceptSearchService, SearchConverterService}
-import no.ndla.conceptapi.service.{ReadService, WriteService}
+import no.ndla.conceptapi.service.search.{DraftConceptSearchService, SearchConverterService}
+import no.ndla.conceptapi.service.{ConverterService, ReadService, WriteService}
 import org.json4s.{DefaultFormats, Formats}
-import org.scalatra.swagger.{ResponseMessage, Swagger, SwaggerSupport}
-import org.scalatra.{Created, NotFound, Ok}
+import org.scalatra.Ok
+import org.scalatra.swagger.{Swagger, SwaggerSupport}
 
 import scala.util.{Failure, Success}
 
-trait ConceptController {
-  this: WriteService with ReadService with User with ConceptSearchService with SearchConverterService =>
-  val conceptController: ConceptController
-
-  class ConceptController(implicit val swagger: Swagger) extends NdlaController with SwaggerSupport with LazyLogging {
+trait DraftConceptController {
+  this: WriteService
+    with ReadService
+    with User
+    with DraftConceptSearchService
+    with SearchConverterService
+    with DraftNdlaController
+    with ConverterService =>
+  val draftConceptController: DraftConceptController
+
+  class DraftConceptController(implicit val swagger: Swagger)
+      extends DraftNdlaControllerClass
+      with SwaggerSupport
+      with LazyLogging {
     protected implicit override val jsonFormats: Formats = DefaultFormats
-    private val conceptId =
-      Param[Long]("concept_id", "Id of the concept that is to be returned")
-
     val applicationDescription = "This is the Api for concepts"
```

Comment:
Kanskje ta med draft her?
```suggestion
    val applicationDescription = "This is the Api for concept drafts"
```